bigquery_etl/query_scheduling/task.py

```diff
@@ -74,6 +77,49 @@ def validate_execution_delta(self, attribute, value):
                 "Timedeltas should be specified like: 1h, 30m, 1h15m, 1d4h45m, ..."
             )
 
+    @schedule_interval.validator
+    def validate_schedule_interval(self, attribute, value):
+        """
+        Validate the schedule_interval format.
+
+        Schedule intervals can be either in CRON format or one of:
+        @once, @hourly, @daily, @weekly, @monthly, @yearly
+        or a timedelta []d[]h[]m
+        """
+        if value is not None and not is_schedule_interval(value):
+            raise ValueError(f"Invalid schedule_interval {value}.")
+
+
+# Know tasks in telemetry-airflow, like stable table tasks
+# https://github.com/mozilla/telemetry-airflow/blob/master/dags/copy_deduplicate.py
+EXTERNAL_TASKS = {
+    TaskRef(
+        dag_name="copy_deduplicate",
+        task_id="copy_deduplicate_main_ping",
+        schedule_interval="0 1 * * *"
+    ): ["telemetry_stable.main_v4"],
+    TaskRef(
+        dag_name="copy_deduplicate",
+        task_id="bq_main_events",
+        schedule_interval="0 1 * * *"
+    ): ["telemetry_derived.main_events_v1"],
+    TaskRef(
+        dag_name="copy_deduplicate",
+        task_id="events_events",
+        schedule_interval="0 1 * * *"
+    ): ["telemetry_derived.events_events_v1"],
+    TaskRef(
+        dag_name="copy_deduplicate",
+        task_id="baseline_clients_last_seen",
+        schedule_interval="0 1 * * *"
+    ): ["*.baseline_clients_last_seen*"],
+    TaskRef(
+        dag_name="copy_deduplicate",
+        task_id="copy_deduplicate_all",
+        schedule_interval="0 1 * * *"
+    ): ["*_stable.*"]
+}
+
```

Comment:
This maps tables that are referenced in a query to the corresponding telemetry-airflow copy_deduplicate task it depends on. Matching tables are specified as patterns.

We might want to have a config file for this in the future, but having this in code will do for now.